helm/hpcc/templates/network.yaml

```diff
@@ -157,6 +157,25 @@ spec:
         matchLabels:
           app: eclservices
 ---
+{{ range .Values.esp -}}
+{{- if has .name $.Values.global.visibilities.exposedServices }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ .name }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ .name }}
+  ingress:
+{{- if .service.ingress }}
+{{ toYaml .service.ingress | indent 2}}
+{{- else }}
+{{ toYaml $.Values.global.visibilities.ingress | indent 2}}
+{{- end }}
```

Comment:
minor: it's worth trying to indent the template language structure for readability.
.e.g the above can become:
```
{{- if .service.ingress }}
 {{- toYaml .service.ingress | nindent 2 }}
{{- else }}
 {{- toYaml $.Values.global.visibilities.ingress | nindent 2 }}
{{- end }}
```

Reply:
fixed